build.rs

```diff
@@ -0,0 +1,11 @@
+fn main() {
+    let std_enabled = std::env::var("CARGO_FEATURE_STD").is_ok();
+    let libm_enabled = std::env::var("CARGO_FEATURE_LIBM").is_ok();
+
+    if std_enabled == libm_enabled {
+        panic!(
```

Comment:
I don't think this is necessary, this could be simplified and written in lib.rs as something like (this is an example from my own code):

```rs
#[cfg(not(any(
    feature = "size_64x32",
    feature = "size_64x64",
    feature = "size_128x128"
)))]
compile_error!(
    "A display size feature must be enabled. Choose one of: size_64x32, size_64x64, size_128x128"
);
```